launcher/ui/pages/instance/WorldListPage.cpp

```diff
@@ -373,11 +373,11 @@ bool WorldListPage::isWorldSafe(QModelIndex)
     return !m_inst->isRunning();
 }
 
-bool WorldListPage::worldSafetyNagQuestion()
+bool WorldListPage::worldSafetyNagQuestion(const QString &actionType)
 {
     if(!isWorldSafe(getSelectedWorld()))
     {
-        auto result = QMessageBox::question(this, tr("Copy World"), tr("Changing a world while Minecraft is running is potentially unsafe.\nDo you wish to proceed?"));
+        auto result = QMessageBox::question(this, tr("%1").arg(actionType), tr("Changing a world while Minecraft is running is potentially unsafe.\nDo you wish to proceed?"));
```

Comment:
as per my other comment, this could be just `actionType` instead of using a `tr` i believe